central/notifiers/cscc/cscc.go

```diff
@@ -158,6 +158,9 @@ func (c *cscc) Close(ctx context.Context) error {
 
 // AlertNotify takes in an alert and generates the notification
 func (c *cscc) AlertNotify(ctx context.Context, alert *storage.Alert) error {
+
+	_, _, _ = c.InitFinding(ctx, alert)
```

Comment:
Use the findingID and finding returned by InitFinding here, and you can remove the code below until
`return retry.WithRetry...` as it is same as InitFinding

```
findingID, finding, err := InitFinding(ctx, alert)
  if err != nil {
      return nil, err
  }
  return retry.WithRetry(...)
```

Reply:
done